README.md

```diff
@@ -66,121 +71,83 @@ This approach dramatically reduces latency and reduces load on the OP's servers.
 
-```json
-{
-   "id_token":{
-      "acr":null,
-      "vp":{
-         "credential_types":[
-            "https://www.w3.org/2018/credentials/examples/v1/IDCardCredential"
-         ]
-      }
-   }
-}
-```
+Note that above claims have to be distinguished from `vp` or `vc` claims as defined in [JWT proof format](https://www.w3.org/TR/vc-data-model/#json-web-token). `vp` or `vc` claims contain those parts of the standard verifiable credentials and verifiable presentations where no explicit encoding rules for JWT exist. They are not meant to include complete verifiable credentials or verifiable presentations objects which is the purpose of the four claims defined in this specification.
 
-### vc in id_token
+This table shows the different combinations of covered by the claims defined in this specificaiton.
 
-A Verifiable Credential embedded in an ID Token is requested by adding a element `vc` to the `id_token` top level element of the `claims` parameter. This element must contain a `credential_types` sub element as defined above.
+|  | vc_jwt | vp_jwt | vc_ldp | vp_ldp
+|:----------------|:---------------|:---------------|:---------|:--------------------------|
+| Object included in the claim | verifiable credential | verifiable presentation | verifiable credential | verifiable presentation 
+| Proof format on the object| JWT | JWT | LD-Proof | LD-Proof
 
-### vp_token
 
-A VP Token is requested by adding a new top level element `vp_token` to the `claims` parameter. This element contains the following sub elements:
+## Requesting W3C Verifiable Credential Objects 
 
-`credential_types`
-Object array containing definitions of credential types the RP wants to obtain along with an (optional) definition of the claims from the respective credential type the RP is requesting. Each of those object has the following fields:
+### Requesting W3C Verifiable Credential Objects using the `claims` parameter
 
-* `type` - String denoting a credential type
+The next section illustrates how the `claims` parameter can be used for requesting verified presentations. It serves as a starting point to drive discussion about this aspect. There are other candidate approaches for this purpose. They will be evaluated as this draft evolves. 
 
-* `claims` - An object determining the claims the RP wants to obtain using the same notation as used underneath `id_token`. 
+#### Requesting Verifiable Presentations
 
-`format`
-String designating the VP format. Predefined values are `jwt` and `json-ld`.
+A Verifiable Presentation embedded in an ID Token is requested by adding a element `vp_jwt` or `vp_ldp` to the `id_token` top level element of the `claims` parameter. This element must contain the following element:
 
-`proof_format`
-[TBD]
+`credential_types`
+A string array containing a list of VC credential types the RP asks for. The OP shall respond with a presentation containing one credential of one of the listed types. 
```

Comment:
definition for "userinfo" is missing

Reply:
as in [https://openid.net/specs/openid-connect-core-1_0.html#ClaimsParameter](https://openid.net/specs/openid-connect-core-1_0.html#ClaimsParameter)?

Reply:
I think it should be "The OP shall respond with a presentation containing credentials of the listed types."

As the text states now, it is misleading that array is how a client asks for any one​ of these types to be satisfied (not all​ of them)